08_files_lab/01.py

```diff
@@ -1,6 +1,18 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+"""
+The program receives two files as input
+and writes the content of the first file to the second one
+"""
+import sys
+
+if len(sys.argv) != 3:
+    print "Usage: 01.py <firstFile> <secondFile>"
+    sys.exit()

```

Comment:
Better to write the error message inside `sys.exit()`. It prints the error to stderr AND returns an error code to the operating system.